lib/managers/files.js

```diff
@@ -613,10 +615,16 @@ Files.prototype.uploadFile = function(parentFolderID, filename, content, options
 		options = {};
 	}
 
+	var formOptions = {};
+	if (options && options.hasOwnProperty('content_length')) {
+		formOptions.knownLength = options.content_length;
+		delete options.content_length;
```

Comment:
What does this do?

Reply:
This gets the content_length and sets it to the key `knownLength` so the request library understands this is the length. Then it deletes it from options that are passed to create headers, body, etc.

Reply:
```suggestion
		// Delete content_length from options so it's not added to the request object later
		delete options.content_length;
```

Reply:
@sujaygarlanka I think that makes sense.  I added a comment to reflect that.  Did I capture it correctly?